test/support/utils.rb

```diff
@@ -18,7 +18,7 @@ def assert_line_num(expected)
 
     def assert_line_text(expected)
       @queue.push(Proc.new {
-        assert_match(expected, @last_backlog[2..].join, create_message("Expected to include #{expected}"))
+        assert_match(expected, @last_backlog[1..].join, create_message("Expected to include #{expected}"))
```

Comment:
I'm curious about the first element of `@last_backlog`. is it the input value?

Reply:
Yes, it's input value such as `(rdbg) step`.

Reply:
I see. thanks for answering